### DISCUSSION TITLE
DONT DOWNGRADE OR UNBLOCK YOUR SCHOOL CHROMEBOOK.

### DISCUSSION THREAD

**Discussion by pl4udible** [3kh0/ext-remover#214] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Reccomended restart your chromebook or switch the ingot back here are thee main reasons on why i don't want you all to do it

. You will end up looking stupid - If a teacher finds out, your gonna have a no electronics record on your reputation
, Downgrading your chromebook to avoid access from managed authorisation is a criminal offence (up to 6 months)
. Just buy a new one, if your parents refuse, just quit and stick with whats left

**Comment by synapt0** [comment]
ill use ingot anyways man

**Comment by 3kh0** [comment]
I have never heard of anyone getting arrested or having something on their record for that

**Comment by JelloFruitAlt** [comment]
why tf would they arrest you lol unless ur actually doing something criminal on ur computer or not doing ur work they really have no reason to bust you for it

**Comment by obkduishinijf** [comment]
my school wants us to downgrade because there was some sort of major flaw with v106 🤷

**Reply by Aka-but-nice** [reply]
Your actually good

**Reply by Aka-but-nice** [reply]
Your living ingot
